src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationMiddleware.cs

```diff
@@ -46,9 +45,14 @@ public class OpenIdConnectAuthenticationMiddleware : AuthenticationMiddleware<Op
         {
             _logger = loggerFactory.CreateLogger<OpenIdConnectAuthenticationMiddleware>();
 
+            if (string.IsNullOrEmpty(Options.SignInScheme))
+            {
+                Options.SignInScheme = externalOptions.Options.SignInScheme;
+            }
```

Comment:
Should you throw if `SignInScheme` is null or empty. I see this check in some middlewares and some lack. 

https://github.com/aspnet/Security/blob/e3b457bac7693818786996a9f5acaef40b02ed1c/src/Microsoft.AspNet.Authentication.Twitter/TwitterAuthenticationMiddleware.cs#L74